source/includes/_role_categories.md

```diff
@@ -0,0 +1,266 @@
+
+### Path Parameters
+
+Parameter | Type | Description
+--------- | ---- | -----------
+id | integer | id for the desired role_categories
+
+## Create Role Category
+
+```javascript
+fetch('https://core.eventtia.com/v1/role_categories/', {
+  method: 'POST',
+  headers: {
+    'Authorization': '<your token>',
+  },
+  body: {
+  {
+    data: {
+        type: "role_categories",
+        attributes: {
+            name: "grupo 5"
+        }
+    }
+  }
+  }
+}
+```
+> Make sure you replace &lt;your token&gt; with the JWT you get when you authenticate. 
+
+> Example of a successful (200) response:
+
+```http
+
+HTTP/1.1 200 OK
+{
+  "data": {
+    "id": "6",
+    "type": "role_categories",
+    "attributes": {
+        "id": 6,
+        "name": "grupo 5"
+    }
+  }
+}
+```
+>Example of Unprocessable Entity (422) response:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+```
+
+This endpoint create an role category and return it
+
+### HTTP Request
+
+`POST /v1/role_categories/`
+
+### Body Parameters
+
+Parameter | Type | Description
+--------- | ---- | -----------
+name | string | The Role category name.
+
+## Update Role Category
+
+```javascript
+fetch('https://core.eventtia.com/v1/role_categories/<id>', {
+  method: 'PUT',
+  headers: {
+    'Authorization': '<your token>',
+  },
+  body: {
+    "data": {
+        "type": "roles",
+        "attributes": {
+            "name": "grupo 10"
+        }
+    }
+  }
+}}
+
+```
+
+> Make sure you replace &lt;your token&gt; with the JWT you get when you authenticate. 
+
+> Make sure you replace &lt;id&gt; for the role_category to update. 
+
+> Example of a successful (200) response:
+
+```http
+HTTP/1.1 200 OK
+{
+  "data": {
+    "id": "5",
+    "type": "role_categories",
+    "attributes": {
+        "id": 5,
+        "name": "grupo 10"
+    }
+  }
+}
+```
+>Example Not Found (404) response: 
+
+```http
+HTTP/1.1 404 Not Found
+{
+  "message": {
+      "error": "code: 128"
+  }
+}
+```
+
+This endpoint update an role category and return it
+
+### HTTP Request
+
+`PUT /v1/role_categories/:id`
+
+### Path Parameters
+
+Parameter | Type | Description
+--------- | ---- | -----------
+id | integer | The id for the desired role_category
+
+### Body Parameters
+
+Parameter | Type | Description
+--------- | ---- | -----------
+name | string | The role category name.
+
+## Destroy Role Category
+```javascript
+fetch('https://core.eventtia.com/v1/role_categories/<id>', {
+  method: 'DELETE',
+  headers: {
+    'Authorization': '<your token>',
+  },
+})
+```
+
+> Make sure you replace &lt;your token&gt; with the JWT you get when you authenticate. 
+
+> Make sure you replace &lt;id&gt; for the role category to destroy. 
+
+>Example of a successful (200) response:
+
+```http
+HTTP/1.1 200 OK
+{
+  "data": {
+    "id": "5",
+    "type": "role_categories",
+    "attributes": {
+        "id": 5,
+        "name": "grupo 10"
+      }
+  }
+}
+```
+
+This endpoint destroy a role category and return it
```

Comment:
This endpoint destroys a role category and returns it

Reply:
done